### ISSUE TITLE
Current master branch does not build: incorrect pydcs dependency

### ISSUE THREAD

**Issue by tomshackell** [dcs-liberation/dcs_liberation#95]
It specifies:
`pydcs>=0.9.10`
in the `requirements.txt`, however, it does not actually run with that version:
```
Traceback (most recent call last):
  File "C:/Dev/dcs_liberation_2.0/qt_ui/main.py", line 12, in <module>
    from qt_ui import uiconstants
  File "C:\Dev\dcs_liberation_2.0\qt_ui\uiconstants.py", line 7, in <module>
    from game.event import UnitsDeliveryEvent, FrontlineAttackEvent
  File "C:\Dev\dcs_liberation_2.0\game\__init__.py", line 1, in <module>
    from .game import Game
  File "C:\Dev\dcs_liberation_2.0\game\game.py", line 3, in <module>
    from game.db import REWARDS, PLAYER_BUDGET_BASE, sys
  File "C:\Dev\dcs_liberation_2.0\game\db.py", line 21, in <module>
    from game.factions.china_2010 import China_2010
  File "C:\Dev\dcs_liberation_2.0\game\factions\china_2010.py", line 79, in <module>
    "jtac_unit": WingLoong_I
NameError: name 'WingLoong_I' is not defined
```
This is because `WingLoong_I` is only defined in the most current version of `PyDCS`, it is not available in the `0.9.10` release, see:
`https://github.com/pydcs/dcs/blob/v0.9.10/dcs/planes.py`

Compared to:
`https://github.com/pydcs/dcs/blob/master/dcs/planes.py`

**Comment by Khopa** [comment]
Yes, there is no release of pydcs on pypy with the Syria map currently. 
And i haven't found a reliable way to point to the github repo version with pip.

Doing this in requirements.txt : 

```pydcs>=git+https://github.com/khopa/dcs@syria_map_update```

or this : 

```pydcs>=git+https://github.com/pydcs/dcs@master```

Just doesn't work for me, pip will just retrieve the latest pydcs 0.9.10...

So you have to retrieve pydcs manually. Download pydcs source, and copy the "dcs" subdirectory to your Python site-packages directory and it should work for now.

**Comment by parithon** [comment]
What if you pulled pydcs in as a submodule. Then change all your import lines for dcs to `from pydcs import dcs` and include the pydcs in your `PYTHONPATH` environment variable:

![image](https://user-images.githubusercontent.com/8602418/90968526-051f6580-e4a2-11ea-87dc-d35afcf0a914.png)

**Comment by Khopa** [comment]
@parithon 
Thanks merged on develop, this should make it easier.

@tomshackell 
Can you try again with the latest develop branch and the submodule thing ?

**Comment by tomshackell** [comment]
I've not been able to get it to work. I've downloaded pydcs into .\pydcs inside the project root and I've added the PYTHONPATH to the run configuration as shown above but it doesn't work for me.

EDIT: Okay I was finally able to get it to work by just copy pasting the dcs directory from pydcs into the project root.

**Comment by walterroach** [comment]
Instead of manually adding the pydcs folder to your PYTHONPATH, you can simply run `python setup.py develop` or `python setup.py install` on pydcs/setup.py to add it to your namespace.  

It appears that version 0.11.0 is now available on pypi though- perhaps it would be better just to add `pydcs>=0.11.0` to the liberation requirements.txt at this point?

**Comment by DanAlbert** [comment]
We send changes to pydcs fairly often, so I doubt we'll get away from the submodule any time soon.

The develop branch doesn't have the same problem. It'll be fixed in master when 2.2 releases.
